duckduckgo/iOS
# Add bundler configuration to limit Gemfile.lock noise

Task/Issue URL: https://app.asana.com/0/414235014887631/1203783226443717/f
CC: @loremattei

Description:
Fix bundler platform to ruby to avoid adding various platforms to Gemfile.lock.
Store gems in a local directory (vendor/bundle) to gain immunity from different fastlane versions being used in other repositories.

Steps to test this PR:

1. Ensure that you’ve followed instructions in [Install Fastlane](https://app.asana.com/0/1202500774821704/1203766784006610/f).
2. call bundle install.
3. verify that Gemfile.lock file has no changes (if you’re on a different bundler version than  2.4.4, you may get a 1-line change to the version).
4. call bundle exec fastlane actions to list available fastlane actions and thus verify that fastlane works.

—

Internal references:

[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
When ready for review, remember to post the PR in MM
